src/ecs/resources/input.rs

```diff
@@ -22,14 +22,52 @@ impl<'a> Iterator for PressedKeysIterator<'a> {
     }
 }
 
+/// Iterator that iterates through all currently pressed down mouse buttons
+pub struct PressedMouseButtonsIterator<'a> {
```

Comment:
It's a non-breaking change as it's not possible to construct a PressedKeysIterator in user code